templates/unit.erb

```diff
@@ -1,9 +1,11 @@
 [Unit]
 Description=Apache Kafka server (<%= (@service_name.split(/-/)[1] and @service_name.split(/-/)[1].capitalize) or 'broker' -%>)
 Documentation=http://kafka.apache.org/documentation.html
+<% if @service_requires_zookeeper %>
```

Comment:
please add spec tests that ensure zookeeper is/isn't present for producer,mirror,consumer,broker